libs/platform/src/lib/components/button/button.component.ts

```diff
@@ -11,13 +11,24 @@ export type ButtonType =
     | 'transparent'
     | 'emphasized'
     | 'menu';
+
 @Component({
     selector: 'fdp-button',
     templateUrl: './button.component.html',
     styleUrls: ['./button.component.scss']
 })
 export class ButtonComponent extends BaseComponent implements AfterViewInit {
 
+    /** Position of glyph related to text */
+    @Input()
+    public glyphPosition = 'before';
```

Comment:
Add `GlyphPosition` type here but import it from core.
```
public glyphPosition: GlyphPosition = 'before';
```

Reply:
updated